cetmix_tower_yaml/tests/test_server_template.py

```diff
@@ -0,0 +1,16 @@
+from odoo.tests import TransactionCase
+
+
+class TestTowerServerTemplate(TransactionCase):
+    def setUp(self, *args, **kwargs):
+        super().setUp(*args, **kwargs)
+
+        self.ServerTemplate = self.env["cx.tower.server.template"]
+
+    def test_server_template_ssh_password_mask(self):
```

Comment:
Add additional tests to cover edge cases, such as missing `ssh_password` or using an empty value.

Reply:
SSH password is an optional field so it should'n be an issue. However the idea to add additional tests looks absolutely reasonable.